src/darwin/Framework/CHIP/MTRCommissionableBrowserDelegate.h

```diff
@@ -26,12 +27,12 @@ MTR_NEWLY_AVAILABLE
 /**
  * Tells the delegate the commissionable manager discovered a device while scanning for devices.
  */
-- (void)didDiscoverCommissionable:(MTRCommissionableBrowserResult *)device;
+- (void)didDiscoverCommissionable:(MTRDeviceController *)controller device:(MTRCommissionableBrowserResult *)device;
 
 /**
  * Tells the delegate a previously discovered device is is no longer available.
  */
-- (void)commissionableUnavailable:(MTRCommissionableBrowserResult *)device;
+- (void)commissionableUnavailable:(MTRDeviceController *)controller device:(MTRCommissionableBrowserResult *)device;
```

Comment:
```suggestion
- (void)controller:(MTRDeviceController *)controller noLongerSeesCommissionableDevice:(MTRCommissionableBrowserResult *)device;
```
or something?  Maybe check on Slack for better naming for this one....

Reply:
I end up naming those `did[Find|Remove]CommissionableDevice`. I have taken inspiration from https://developer.apple.com/documentation/foundation/nsnetservicebrowserdelegate?language=objc